WorkflowAdminBundle/Controller/Admin/ParametersController.php

```diff
@@ -0,0 +1,141 @@
+<?php
+
+namespace OpenOrchestra\WorkflowAdminBundle\Controller\Admin;
+
+use Sensio\Bundle\FrameworkExtraBundle\Configuration as Config;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+use OpenOrchestra\Backoffice\Security\ContributionActionInterface;
+use OpenOrchestra\BackofficeBundle\Controller\AbstractAdminController;
+use OpenOrchestra\ModelInterface\Model\StatusInterface;
+use OpenOrchestra\ModelInterface\Event\StatusEvent;
+use OpenOrchestra\ModelInterface\StatusEvents;
+
+/**
+ * Class ParametersController
+ */
+class ParametersController extends AbstractAdminController
+{
+    /**
+     * @param Request $request
+     *
+     * @Config\Route("/workflow-parameters/form", name="open_orchestra_workflow_admin_parameters_form")
+     * @Config\Method({"GET", "POST"})
+     *
+     * @return Response
+     */
+    public function formAction(Request $request)
+    {
+        $this->denyAccessUnlessGranted(ContributionActionInterface::EDIT, StatusInterface::ENTITY_TYPE);
+
+        $statusCollection = $this->get('open_orchestra_model.repository.status')->findNotOutOfWorkflow();
+        $data = array(
+            'statuses' => $statusCollection,
+            'labels'   => array(
+                $this->get('translator')
+                    ->trans('open_orchestra_workflow_admin.status.initial_state'          , array(), 'interface'),
+                $this->get('translator')
+                    ->trans('open_orchestra_workflow_admin.status.translation_state'      , array(), 'interface'),
+                $this->get('translator')
+                    ->trans('open_orchestra_workflow_admin.status.published_state'        , array(), 'interface'),
+                $this->get('translator')
+                    ->trans('open_orchestra_workflow_admin.status.auto_publish_from_state', array(), 'interface'),
+                $this->get('translator')
+                    ->trans('open_orchestra_workflow_admin.status.auto_unpublish_to_state', array(), 'interface'),
+            )
+        );
+
+        $form = $this->createForm('oo_workflow_parameters', $data, array(
+            'action' => $this->generateUrl('open_orchestra_workflow_admin_parameters_form')
+        ));
+
+        $updatedStatusIndexes = array();
+        if ($request->isMethod('POST')) {
+            $updatedStatusIndexes = $this->getStatusIndexesToUpdate(
```

Comment:
@alavieille : because this stuff determine wich statuses are requested to be update comparing the request and the status collection from the base. Il the job is done in the ``handleform``, the collection is already updated with the request, and it is impossible to find the updated statuses. The only way I've found is to do this before the request is merged into the form, ie before ``handleRequest``